### PR TITLE
Make ESLint Work

### PR DESCRIPTION
**Pull request by drewpc**
ESLint wasn't properly listing all files in `react-server-cli` because if a missing glob pattern.  Additional cleanup was to add a `lint` run script to each package.json file and fix a dependency issue with `react-server-module-tagger`

### CODE REVIEW

File: packages/react-server-cli/src/commands/start.js [hunk 1]
```diff
@@ -13,53 +13,44 @@ import logProductionWarnings from "../logProductionWarnings";
...
-		jsUrl,
-		httpsOptions,
-		longTermCaching,
-		customMiddlewarePath,
-	} = options;
 
-	const {serverRoutes, compiler} = compileClient(options);
+// returns a method that can be used to stop the server. the returned method
+// returns a promise to indicate when the server is actually stopped.
+const serverToStopPromise = (server) => {
```
Comment (gigabo) [inline packages/react-server-cli/src/commands/start.js]: Ugh, thanks for going through all this!

**Comment by gigabo** [comment]
Thanks for tracking down the problem in `react-server-cli` and getting this all cleaned up @drewpc!

**Comment by drewpc** [comment]
No problem!  It's all happy now:

```
$ asini run lint
Successfully ran npm script 'lint' in packages:
- babel-plugin-react-server
- generator-react-server
- react-server-cli
- react-server-core-middleware
- react-server-data-bundle-cache
- react-server-examples
- react-server-gulp-module-tagger
- react-server-module-tagger
- react-server-test-pages
- react-server-website
- react-server
```

**Comment by roblg** [comment]
🚀!

Does this depend on #809 to be merged first? If not, let's merge!

**Comment by drewpc** [comment]
@roblg Nope, no dependencies at all.  The only reason that `start.js` was modified is because ESLint complained about functions being used before they were defined in that file.

**Pull request merged**